### ISSUE TITLE
Feature request -  SELinux Alerts

### ISSUE THREAD

**Issue by michaellindman** [dje4321/LinuxStatus#1]
Added support for an option to check and receive SELinux AVC alert notifications within the programs output for SELinux enabled systems.

**Comment by dje4321** [comment]
Good idea but hard to implement due to the need for root access. Probably wont add it but might be something i work on during down time
